cli/cmd/trigger_delivery.go

```diff
@@ -170,39 +171,36 @@ func doTriggerDelivery(deliveryInputData deliveryStruct) error {
 
 func doTriggerDeliveryPreRunCheck(deliveryInputData deliveryStruct) error {
 	trimmedImage := strings.TrimSuffix(*deliveryInputData.Image, "/")
-	deliveryInputData.Image = &trimmedImage
 
-	if deliveryInputData.Tag == nil || *deliveryInputData.Tag == "" {
-		*deliveryInputData.Image, *deliveryInputData.Tag = docker.SplitImageName(*deliveryInputData.Image)
-	}
-	return docker.CheckImageAvailability(*deliveryInputData.Image, *deliveryInputData.Tag, nil)
+	image, tag := docker.SplitImageName(trimmedImage)
+	sanitizedImageName := fmt.Sprintf("%s:%s", image, tag)
+	deliveryInputData.Image = &sanitizedImageName
```

Comment:
does this have an effect?

Reply:
the `SplitImageName` function checks if there is already a tag defined in the provided image, and defaults it to `latest`, if there is none. Therefore I decided to keep using it

Reply:
i mean the assignment in line 177

Reply:
nope, that did not have an effect. has been removed in the meantime